pkg/cmd/release/create/create.go

```diff
@@ -76,7 +77,13 @@ func NewCmdCreate(f *cmdutil.Factory, runF func(*CreateOptions) error) *cobra.Co
 			# upload a release asset with a display label
 			$ gh release create v1.2.3 '/path/to/asset.zip#My display label'
 		`),
-		Args: cobra.MinimumNArgs(1),
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) > 0 {
+				return nil
+			}
+
+			return &cmdutil.FlagError{Err: errors.New("could not create: no tag name provided")}
+		},
```

Comment:
It seems we have a couple of those custom messages already. Should we create something like:
```go
func MinimumArgs(n int, msg string) cobra.PositionalArgs {
        if msg == "" {
                return cobra.MinimumNArgs(1)
        }

  	return func(cmd *Command, args []string) error {
		if len(args) < n {
			return &cmdutil.FlagError{Err: errors.New(msg)}
		}
		return nil
	}
}
```
and we could use like:
``` go
Args: cmdutil.MinimumArgs(1, "could not create: no tag name provided")
```

Reply:
That's a good idea; happy for that to be a follow-up, though.